python/doc/examples/reliability_sensitivity/design_of_experiments/plot_smolyak_merge.py

```diff
@@ -192,12 +183,9 @@ def computeNumberOfSmolyakNodes(level, epsilon_a, epsilon_r):
 graph = ot.Graph("Sensitivity to merge", "Level", "Size decrease", True)
 cloud = ot.Cloud(level_list, size_decrease_list)
 graph.add(cloud)
-view = otv.View(
-    graph,
-    figure_kw={"figsize": (4.0, 3.0)},
```

Comment:
That is not the purpose of this commit. The figure size setting must be kept as in the original code: only the legend setting must be updated here.